src/main/java/com/qualitestgroup/java/assessment/framework/pages/DemoPage.java

```diff
@@ -24,6 +36,30 @@ public DemoPage(WebDriver driver) {
         global.setProdURL();
     }
 
+    public boolean angularCheckBox() {
+        boolean isStep = false;
+        page.navigateTo(global.getProdUrl());
+        page.scrollToElement(elementCard, 30);
+        page.clickElement(elementCard, 20);
+        page.clickElement(checkBx, 20);
+        page.clickElement(expandbutton, 20);
+        page.scrollToElement(angularChckBx, 20);
+        page.clickElement(angularChckBx, 20);
+
+        String value[] = page.getAttributeValueArray(angularChckBx + svcElement, 20, "class");
+        if (value[3].equals("check")) {
+
+            isStep = true;
+        }
```

Comment:
Why not just return false if it does not equal "check"? It would be wasted execution time to continue after a fail.